integrationTests/vm/esdt/multi-transfer/multiTransferCommon.go

```diff
@@ -0,0 +1,283 @@
+package multitransfer
+
+import (
+	"encoding/hex"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	testVm "github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/arwen"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/esdt"
+	vmFactory "github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/ElrondNetwork/elrond-go/testscommon/txDataBuilder"
+	"github.com/ElrondNetwork/elrond-go/vm"
+	"github.com/stretchr/testify/require"
+)
+
+const NR_ROUNDS_CROSS_SHARD = 15
```

Comment:
these 2 consts should be made unexported or have a comment
bad naming: 1. use camelCase, 2. do not use CAPS, 3. change NR->NUM